app/src/main/java/com/nervousfish/nervousfish/activities/MainActivitySorter.java

```diff
@@ -105,6 +105,13 @@ public void onItemClick(final AdapterView<?> parent, final View view, final int
 
         });
 
+        if (contactsByNameListAdapter.isEmpty()) {
+            mainActivity.findViewById(R.id.view_flipper_sorter_main).setVisibility(View.GONE);
+            mainActivity.findViewById(R.id.no_users_screen).setVisibility(View.VISIBLE);
+        } else {
+            mainActivity.findViewById(R.id.view_flipper_sorter_main).setVisibility(View.VISIBLE);
+            mainActivity.findViewById(R.id.no_users_screen).setVisibility(View.GONE);
+        }
```

Comment:
This seems to be duplicate code with my next comment, can you somehow resolve that 😇